extensions/smallrye-opentracing/runtime/pom.xml

```diff
@@ -110,6 +110,19 @@
                     </annotationProcessorPaths>
                 </configuration>
             </plugin>
+            <!-- The filter classes needs to be indexed -->
+            <plugin>
```

Comment:
I think we can forgo this by generating a JaxrsProviderBuildItem with the provider you added

Reply:
Any examples of that I can look at? I'm still a newborn when it comes to extensions :-)

Reply:
Any of the usages of https://github.com/quarkusio/quarkus/blob/main/extensions/resteasy-classic/resteasy-common/spi/src/main/java/io/quarkus/resteasy/common/spi/ResteasyJaxrsProviderBuildItem.java should be just fine 🙂

Reply:
Looking at examples I think this should work:
```
} else if (capabilities.isPresent(Capability.RESTEASY_REACTIVE)) {
            customResponseFilters.produce(new CustomContainerResponseFilterBuildItem(
                    QuarkusSmallRyeTracingStandaloneContainerResponseFilter.class.getName()));
            dynamicFeatures.produce(new DynamicFeatureBuildItem(QuarkusSmallRyeTracingDynamicFeature.class.getName()));
// new...
            providers.produce(
                    new ResteasyJaxrsProviderBuildItem(
                            QuarkusSmallRyeTracingStandaloneContainerResponseFilter.class.getName()));
}
```

But alas its not being registered

Reply:
Ah darn. Sorry, the build item I added above is for RESTEasy Classic

Reply:
https://github.com/quarkusio/quarkus/blob/main/extensions/resteasy-reactive/quarkus-resteasy-reactive-common/spi-deployment/src/main/java/io/quarkus/resteasy/reactive/spi/ContainerRequestFilterBuildItem.java should be the one to uss

Reply:
How about this one:
writerInterceptors.produce(
                    new WriterInterceptorBuildItem.Builder(
                            QuarkusSmallRyeTracingStandaloneContainerResponseFilter.class.getName()).build());

It works btw

Reply:
Ah, it's an interceptor! So yeah, that should be the one to use!